### PR TITLE
Feature socket connection

### PR DESCRIPTION
**Pull request by JonathanArnould**
Implement socket and a redux middleware to get game position of others users connected.

### CODE REVIEW

File: server/src/config/server-config.ts [hunk 1]
```diff
@@ -1,4 +1,13 @@
 import dotenv from 'dotenv';
+<<<<<<< HEAD
```
Comment (AuxB) [inline server/src/config/server-config.ts]: conflit non résolu

File: client/src/redux/middleware/socketMiddleware.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { Middleware } from 'redux';
```
Comment (AuxB) [inline client/src/redux/middleware/socketMiddleware.ts]: nommage du fichier doit être en kebab case (il me semble qu'on avait statué sur ca)

File: server/src/server.ts [hunk 3]
```diff
@@ -2,6 +2,21 @@ import 'reflect-metadata';
 import dotenv from 'dotenv';
 import { buildSchema } from 'type-graphql';
 import mongoose from 'mongoose';
+<<<<<<< HEAD
```
Comment (AuxB) [inline server/src/server.ts]: conflit non resolu

File: server/src/config/server-config.ts (hunk 1 above)
Comment (lucas240) [inline server/src/config/server-config.ts]: there are conflict not resolve

File: server/src/models/user-model.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import { prop as Property, modelOptions } from '@typegoose/typegoose';
```
Comment (lucas240) [inline server/src/models/user-model.ts]: delete this file, we already have users-model.ts

File: server/src/server.ts (hunk 3 above)
Comment (lucas240) [inline server/src/server.ts]: there are conflict not resolve

File: server/src/resolvers/user-resolver.ts [hunk 5]
```diff
@@ -0,0 +1,55 @@
+/* eslint-disable class-methods-use-this */
```
Comment (lucas240) [inline server/src/resolvers/user-resolver.ts]: delete this file, we already have users-resolver.ts

**Review by AuxB — CHANGES_REQUESTED** [review]

**Review by lucas240 — APPROVED** [review]

**Review by AuxB — APPROVED** [review]

**Pull request merged**